Jenkinsfile

```diff
@@ -1155,3 +1160,14 @@ def tryFinally(Closure main, Closure ... finallies) {
 		throw mainFailure
 	}
 }
+
+void checkNoSourceFilesModifiedOrFail() {
+	echo 'Checking sources for changes'
+	// We are ignoring the failsafe-summary.xml as this file may get modified by integration tests running on
+	//  non-default envs and we don't want to change that at the moment:
```

Comment:
Shouldn't this just be in `.gitignore`?

In fact I'd put `/.empty/*` in gitignore; we should never commit anything in a directory named `.empty` whose only purpose is being empty...

Reply:
I thought about it yes, but there are a couple files in that "empty" directory 🙈 so I thought I'd better keep it as is 😃

Reply:
Ah, ok, I misunderstood. Yes I see `.empty` is not supposed to be empty, but `.empty/directory` is, and `.empty/failsafe-summary.xml` is actually a checked-in file representing an empty test run.

Now I'm wondering why test runs would end up modifying this file... There must be something wrong :/

Reply:
I noticed that the summary file was only modified by the OpenSearch build (so probably non-default builds that run tests ... ). But if you are saying that it should've stayed unchanged, I'll look into why it gets updated.